.env.defaults

```diff
@@ -27,6 +27,8 @@ OG_IMAGE_URL=
 SITE_CANONICAL_URL=https://lbry.tv
 # LOCALE
 DEFAULT_LANGUAGE=en
+# Custom Settings
+# UNSYNCED_SETTINGS='theme dark_mode_times automatic_dark_mode_enabled'
```

Comment:
I guess just keep this, but also include a comment pointing to where these names are defined